### DISCUSSION TITLE
Consider using react-admin for the admin part

### DISCUSSION THREAD

**Discussion by fzaninotto** [amplication/amplication#1119] [Ideas]
Hi, and kudos for the great work so far. We definitely need more low-code solutions to build web apps. The success of solutions like strapi show that developers need tools to stop reinventing the wheel. 

I'm one of the lead developers of an open-source project called react-admin (https://github.com/marmelab/react-admin), which does part of what you're covering with amplication: generating an admin on top of an API in React. We've been working on it for more than 4 years now, and my feedback is that it's already a huge domain. The differences between a basic CRUD and a really usable admin are massive, and the needs of admin users are all different. 

My point is that if your goal is very ambitious, and that you could get there faster by standing on other open-source projects shoulders. We did the same: react-admin is built on top of Redux, react-final-form, and material-ui. 

So what are your thoughts about using an existing admin framework in amplication?

**Comment by yuval-hazaz** [comment]
Hi @fzaninotto 
Impressive work... well done!
We are focusing on the server-side and decided to provide the minimal starting point on the client. I agree that the client-side domain is big, so we would love to find a way to join forces and integrate with existing solutions for the client.

Let's talk 
https://discord.gg/b8MrjU6
